docs/sources/dashboards/build-dashboards/annotate-visualizations/index.md

```diff
@@ -119,6 +119,8 @@ To add a new annotation query to a dashboard, take the following steps:
 
 After you add an annotation, they will still be visible. This is due to the built-in annotation query that exists on all dashboards. This annotation query will fetch all annotation events that originate from the current dashboard, which are stored in Grafana, and show them on the panel where they were created. This includes alert state history annotations.
 
+The built-in annotation query requires the special `-- Grafana --` data source to function, and the data source selection shouldn't be changed. 
```

Comment:
```suggestion
Manual annotations are only supported when you use the `-- Grafana --` data source in the built-in annotation query. You can use another data source, but it will only allow for automated queries using the editor for that data source.
```

Reply:
@sarahzinger & @axelavargas - What about this?

Reply:
Nice, still concise but with more clarity 🏅

Reply:
love it!